wandb/sdk/artifacts/__init__.py

```diff
@@ -0,0 +1,12 @@
+from wandb.sdk.artifacts.local_artifact import Artifact as LocalArtifact
+from wandb.sdk.artifacts.public_artifact import ARTIFACT_FRAGMENT  # noqa: F401
+from wandb.sdk.artifacts.public_artifact import Artifact as PublicArtifact
```

Comment:
Can we call these `DraftArtifact` and `ComittedArtifact`? That gets a little more confusing to me when we log (and therefore commit) a draft artifact, but I don't know that I would know what a "PublicArtifact" is supposed to mean.

Reply:
I agree this naming isn't great. I just followed the existing convention.
We will merge those classes in the next PR, so this doesn't really matter.